src/wayland/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIRWAYLAND_1.2 {
+MIRWAYLAND_1.3 {
```

Comment:
I guess we're breaking ABI? If so, we ought to bump the soname too.

The stanza name for Mir 2.0 should be MIRWAYLAND_2.0.